modules/default.nix

```diff
@@ -1,6 +1,6 @@
 { configuration
 , pkgs
-, lib ? pkgs.lib
+, lib ? extraSpecialArgs.lib or pkgs.lib
```

Comment:
Why is this needed? Can't you just pass `lib` directly as an argument to this file?

Reply:
See my response to your issue comment.